dipy/viz/window.py

```diff
@@ -514,7 +514,7 @@ def _stop_recording_and_close(obj, evt):
             self.iren.Start()
             # Deleting this object is the unique way
             # to close the file.
-            del recorder
+            recorder = None
```

Comment:
In the `_stop_recording_and_close ` shouldn't we have something like: 

    if (recorder):
        recorder.stop()

to prevent crash because of nullref?  
Or is it that that scenario won't happen?

Reply:
From what I understand `self.iren.Start()` is blocking until the user closes the window at which point the ExitEvents are handled before releasing the block. So, it should happen that recorder is None inside `_stop_recording_and_close`. 

But, yeah, we could add the check to be on the safe side because something VTK still surprises me :P.

Reply:
ahaha :-) , @dmreagan told me to do that and I gave the same answer as @MarcCote.

No problem, I will add this test when I will get access to my computer.